src/client/orderbook/OrderbookFullClient.ts

```diff
@@ -201,6 +210,31 @@ export class OrderbookFullClient extends OrderbookReadClient {
     };
   }
 
+  public async listMultipleNftsWithCommission(p: {
+    beneficiary: SuiAddress;
+    commissions: number[];
+    collection: string;
+    ft: string;
+    orderbook: ObjectId;
+    prices: number[];
+    nfts: ObjectId[];
+    sellerSafe: ObjectId;
+    ownerCap: ObjectId;
+  }) {
+    const effects = await this.client.sendTxWaitForEffects(
+      listMultipleNftsWithCommissionTx({
+        ...this.opts,
+        ...p,
+      })
+    );
+
+    return {
+      // undefined if trade not executed instantly
+      trade: effects.created?.find(Boolean)?.reference.objectId,
```

Comment:
It'd be great to also add tests, because this bit won't work as expected and we'd have it proven by tests